go_expvar/datadog_checks/go_expvar/go_expvar.py

```diff
@@ -169,10 +170,13 @@ def parse_expvar_data(self, data, tags, metrics, max_metrics, namespace):
                 try:
                     float(value)
                 except (TypeError, ValueError):
-                    self.log.warning("Unreportable value for path %s: %s", path, value)
+                    self.log.warning("Unreportable value for path %s: %s", actual_path, value)
                     continue
 
                 if count >= max_metrics:
+                    self.log.warning(
+                        "Exceeded maximum allowed metrics (%s) while processing: %s", max_metrics, metric_name
+                    )
```

Comment:
We already call log.warning with the warning call, maybe we can add the info there instead?

Reply:
Or alternatively, treat this as a `debug` log?